app/jobs/discover_runner_job.rb

```diff
@@ -0,0 +1,13 @@
+class DiscoverRunnerJob < ApplicationJob
+  queue_as :discover_runner
+
+  def perform(run)
+    srdc_runner_id = SpeedrunDotCom::Run.runner_id(run.srdc_id)
+    return if srdc_runner_id.nil?
+
+    twitch_login = SpeedrunDotCom::User.twitch_login(srdc_runner_id)
+    return if twitch_login.blank?
+
+    run.update(user: User.find_by(name: twitch_login))
```

Comment:
Oops, never updated this before, but this should be something like
```ruby
run.update(user: User.joins(:twitch).find_by(twitch_users: {name: twitch_login}))
```
now.

Reply:
Worked perfectly!